blocks/library/cover-image/style.scss

```diff
@@ -41,5 +42,44 @@
 	&.components-placeholder {
 		height: inherit;
 	}
+	
+	.button-container {
+		top: 100px;
+		font-family: $default-font;
```

Comment:
Why do we explicitly override the font family?

Reply:
This logic was already being applied in button code. I did some checks and without it, in some themes, we use other fonts that don't look as nice in the button. So, for now, I kept this font family override.